Add osu!idol 2019 Finals: Community Voting Results news post (#2956)
ppy/osu-wiki: news/2019-12-21-osu-idol-2019-finals-community-voting-results.md

---
layout: post
title: "osu!idol 2019 Finals: Community Voting Results"
date: 2019-12-21 12:00:00 +0000
---

> We have concluded our osu!idol contest for this year. Thank you to everyone that was able to participate in the [community voting](https://osu.ppy.sh/home/news/2019-11-19-osu-idol-2019-finals-community-voting)! Time for the results.

Reviewer: ```suggestion
We have concluded our osu!idol contest for this year. Thank you to everyone that was able to participate in the [community voting](https://osu.ppy.sh/community/contests/83)! It's time for the results.
```

![](/wiki/shared/news/2019-11-19-osu-idol-2019-finals-community-voting/banner.jpg)

### Results

> **Congratulations to this year's osu!idol: [iamALieee](https://osu.ppy.sh/users/10093980)!** Their amazing performance of **Arisa Komiya - White First Love** acquired 1,283 votes, earning them first place!

Reviewer: should probably only bold the name. song titles can be italics or somethng
```suggestion
Congratulations to this year's osu!idol: **[iamALieee](https://osu.ppy.sh/users/10093980)**! Their amazing performance of Arisa Komiya's *White First Love* acquired 1,283 votes, earning them first place!
```
same for other 2 contestants

> **Congratulations to this year's osu!idol: [iamALieee](https://osu.ppy.sh/users/10093980)!** Their amazing performance of **Arisa Komiya - White First Love** acquired 1,283 votes, earning them first place!

Reviewer: maybe mention what prizes each of them won?

Author: I added the prizes to each contestant, if that looks too tacky I could list them like in the previous announcement

Reviewer: looks fine 👍

**Coming up next we have [aceredshirt13](https://osu.ppy.sh/users/13736238)** with their performance of **Darren Korb - Setting Sail, Coming Home** taking second with 290 votes!

**In a close third, with 204 votes, is [moof](https://osu.ppy.sh/users/8457202)** with their performance of **niki - Jitter Doll**! Another job well done!

> Links to the performances:

Reviewer: ```suggestion
You can listen to their performances here:
```
